nfs-server.yml

```diff
@@ -13,56 +13,56 @@ spec:
         role: nfs-server
     spec:
       containers:
-      - name: nfs-server
+      - command:
+        - /bin/bash
+        - -c
+        - 'while [ ! -d /workflow-data ] ; do "Waiting for /workflow-data to created"
```

Comment:
formatowanie rozwalone. Czemu ">" zastąpił jest zastąpione quotami?

Reply:
Faktycznie, przepuszczam te yamle przez pythona i na wyjściu potem tak losowo wychodzi to formatowanie, zapomniałem to uwzględnić w przypadku nfs-server.yml'a. 
Już poprawione w kolejnym commicie